src/components/Combobox/__tests__/combobox-tests.jest.js

```diff
@@ -64,6 +64,19 @@ describe("Combobox tests", () => {
       await waitFor(() => expect(screen.getByText(noRes)).toBeInstanceOf(Node));
     });
 
+    it("should support default filter", async () => {
```

Comment:
cool, we have also check for trying to filter something and check if it's working (when there is no default filter? just for checking out we don't have regression)